src/angular-business/header/header/header.component.scss

```diff
@@ -14,47 +14,73 @@ $sbbHeaderHeight: 54px;
   border-bottom: solid 1px $sbbColorSilver;
 
   > nav {
-    margin-left: 50px;
     overflow-x: hidden;
     display: flex;
     flex-grow: 1;
+  }
+
+  &:not(.sbb-header-flexible) > nav {
+    margin-left: pxToRem(50);
 
     > ::ng-deep a,
     > ::ng-deep button {
       flex: 0 0 20%;
     }
   }
+
+  &.sbb-header-flexible > nav {
+    > ::ng-deep a,
+    > ::ng-deep button {
+      margin-right: 1rem;
```

Comment:
should we not use UX predefined dividers?

Reply:
Approved by @Frenggi